Formula/php.rb

```diff
@@ -61,12 +60,7 @@ class Php < Formula
   uses_from_macos "libxml2"
   uses_from_macos "libxslt"
   uses_from_macos "zlib"
-
-  on_macos do
-    # PHP build system incorrectly links system libraries
-    # see https://github.com/php/php-src/pull/3472
-    patch :DATA
```

Comment:
The patch is still needed. Or, at least, some of it is.
```
configure: WARNING: unrecognized options: --enable-phpdbg-webhelper, --with-xmlrpc, --with-os-sdkpath
```

Reply:
If we remove the patch, shouldn't we remove the `--with-os-sdkpath` option too?

Reply:
Not being able to pass `--with-os-sdkpath` is probably breaking the build though.

Reply:
I see, I added the patch back.